hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/block/BlockManagerImpl.java

```diff
@@ -284,6 +287,9 @@ public void close() throws IOException {
       deletedBlockLog.close();
     }
     blockDeletingService.shutdown();
+    if (metrics != null) {
+      ScmBlockDeletingServiceMetrics.unRegister();
```

Comment:
NIT: can we set metrics null after unRegister?

Reply:
Sure.